cmake/CMakeLists.txt

```diff
@@ -849,8 +855,12 @@ if (onnxruntime_USE_CUDA)
   endif()
   list(APPEND onnxruntime_EXTERNAL_LIBRARIES ${ONNXRUNTIME_CUDA_LIBRARIES})
 
-  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_30,code=sm_30") # K series
-  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_50,code=sm_50") # M series
+  # the following compute capabilities are deprecated in CUDA 11 Toolkit
+  if (CUDA_VERSION_MAJOR LESS 11)
+    set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_30,code=sm_30") # K series
+    set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_50,code=sm_50") # M series
+  endif()
```

Comment:
Does it mean it will drop support for Tesla M60 GPU ? We're using this one in our CI build.

Reply:
Yes, M60 support is being deprecated but not dropped yet in CUDA 11.  In CUDA 11, Support for Kepler sm_30 and sm_32 architecture based products is dropped.  sm_35, sm_37 and sm_50 are deprecated. Here is the details: https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#deprecated-features